fgpyo/sam/__init__.py

```diff
@@ -201,9 +201,15 @@
 NO_REF_POS: int = -1
 """The reference position to use to indicate no position in SAM/BAM."""
 
+NO_QUERY_BASES: str = "*"
+"""The string to use for a SAM record with missing query bases."""
+
 NO_QUERY_QUALITIES: array = qualitystring_to_array(STRING_PLACEHOLDER)
 """The quality array corresponding to an unavailable query quality string ("*")."""
 
+OK_BASES: set[str] = {"A", "C", "G", "T", "="}
```

Comment:
suggestion: shall we make this private for now since it's only used here and not intended to be use by outside projects
```suggestion
_OK_BASES: set[str] = {"A", "C", "G", "T", "="}
```